### PR TITLE
add initial cli for setting root and executing. Successfully calls ou…

### PR DESCRIPTION
**Pull request by 0xAustinWang**
…t to the library

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringVar(&rpc, "rpc", "", "RPC endpoint for the command to connect to")
+	rootCmd.PersistentFlags().StringVar(&proposal, "proposal", "p", "Absolute file path containing the proposal to be submitted")
+	rootCmd.PersistentFlags().StringVar(&chainSelector, "selector", "-1", "Chain selector for the command to connect to")
```
Comment (carte7000) [inline cmd/root.go:33]: Are all the tools always going to target a single chain at once?

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
```
Comment (carte7000) [inline cmd/root.go:30]: Should we add unit test for those command? https://gianarb.it/blog/golang-mockmania-cli-command-with-cobra

File: cmd/root.go (hunk 1 above)
Comment (0xAustinWang) [reply cmd/root.go:33]: initially for the cli, yes. This is so we can execute through github actions and utilize the parallelization there

**Comment by 0xAustinWang** [comment]
closed in favor of https://github.com/smartcontractkit/ccip-owner-contracts/pull/15 due to staleness

**Pull request closed without merging**